deployment/deploy.js

```diff
@@ -14,12 +14,39 @@
  *  OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR
  *  PERFORMANCE OF THIS SOFTWARE.
  */
-const Deployer = require('aeproject-lib').Deployer;
+const { Universal, MemoryAccount, Node, Crypto } = require('@aeternity/aepp-sdk');
+const contractUtils = require('../utils/contract-utils');
 
-const deploy = async (network, privateKey, compiler, networkId) => {
-    let deployer = new Deployer(network, privateKey, compiler, networkId)
+const NETWORKS = require('../config/network.json');
+const DEFAULT_NETWORK_NAME = 'local';
 
-    await deployer.deploy("./contracts/ExampleContract.aes")
+const EXAMPLE_CONTRACT_SOURCE = './contracts/ExampleContract.aes';
```

Comment:
Can this path be changed to the actual contract file?

Reply:
did do